app-metrics/prometheus-nginxlog-exporter/prometheus-nginxlog-exporter-1.11.0.ebuild

```
@@ -0,0 +1,64 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+# Project name and version
+DESCRIPTION="Prometheus NGINX log exporter"
+HOMEPAGE="https://github.com/martin-helmich/prometheus-nginxlog-exporter"
+SRC_URI="
+		${HOMEPAGE}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+		https://files.adjust.com/${P}.tar.xz -> ${P}-deps.tar.xz
+"
+
+LICENSE="Apache-2.0 license"
+KEYWORDS="~amd64"
+SLOT="0"
+IUSE="doc"
+
+DEPEND="
+	dev-lang/go
```

```suggestion
	acct-group/nginx
	acct-user/nginx
	dev-lang/go
```
Service needs `nginx` user